### ISSUE TITLE
Focus Indicator in FastPass and Assessment report not displaying correctly

### ISSUE THREAD

**Issue by brocktaylor7** [microsoft/accessibility-insights-web#5067]
## Describe the bug

In both the FastPass and Assessment reports the focus indicator for the target page link is not displaying correctly.

**To Reproduce**

1. Run FastPass or Assessment
2. Export the report as HTML
3. Tab until the Target Page link in the top-right has focus

## Expected behavior

The focus indicator should be a full box around the focused element.

## Screenshots

![image](https://user-images.githubusercontent.com/13286385/149587440-80e5886f-99ef-43ca-be69-0143641c12af.png)

## Context (please complete the following information)

- OS Name & Version: Windows 11 Pro 21H2 Build 22000.376
- AI-Web Version & Environment: 2.29.0 Production
- Browser Version: Microsoft Edge Version 97.0.1072.55 (Official build) (64-bit)
- Target Page: [](http://www.washington.edu/accesscomputing/AU/before.html)

## Are you willing to submit a PR?

Yes

## Did you search for similar existing issues?

Yes
